From 0de79921d70e360de4b2ef3a41af340ef4fc99c0 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Wed, 30 Dec 2020 18:23:13 +0100
Subject: [PATCH 01/21] Fix details in clean Make target when TARGET
 architecture is specified

'rm -rf builddir-$(TARGET)' had no effect after 'rm -rf builddir-*'.

'rm -rf $(d_cross)' had no effect without TARGET (if TARGET is empty,
d_cross=debian/binutils- never exists).

Then, debian/{files,substvars} do not need '-r'.
---
 debian/rules | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/rules b/debian/rules
index b0250c2..a8d1c40 100755
--- a/debian/rules
+++ b/debian/rules
@@ -609,9 +609,9 @@ clean:
 	$(foreach ca,$(CROSS_ARCHS), rm -rf debian/binutils-$(shell echo $(call _gnu_type,$(ca)) | sed 's/_/-/g;s/i586/i686/'){,-dbg}; )
 	rm -rf debian/tmp debian/files* debian/substvars
 	rm -f debian/*.orig debian/*.rej
-	rm -rf $(d_cross) debian/files debian/substvars
+	rm -f debian/files debian/substvars
 ifneq (,$(TARGET))
-	rm -rf builddir-$(TARGET)
+	rm -rf $(d_cross)
 endif
 	for i in debian/*.in; do \
 	    case "$$i" in debian/control*.in) continue; esac; \
-- 
2.30.1

